examples/dmabuf-capture.c

```diff
@@ -506,10 +505,10 @@ static void *vid_encode_thread(void *arg) {
 				goto end;
 			}
```

Comment:
If any of the above branches are hit, the AVPacket will be leaked.

Reply:
Fixed